tools/rush-tools-new.ts

```diff
@@ -0,0 +1,276 @@
+import * as path from "path";
+import * as shelljs from "shelljs";
+import * as semver from "semver";
+
+import {
+    CommandLineParser,
+    CommandLineFlagParameter,
+    CommandLineAction,
+    CommandLineStringListParameter,
+    CommandLineStringParameter
+} from "@microsoft/ts-command-line";
+
+import { RushConfiguration } from "@microsoft/rush-lib";
+
+// Tasks
+
+class ShellTask {
+    constructor(private rushConfiguration: RushConfiguration, private projects: string[]) { }
+
+    public Execute(command: string): void {
+        this.consoleWrite(`Starting: ${command}`);
+
+        const failedProjects: string[] = [];
+        const succeededProjects: string[] = [];
+
+        for (const projectName of this.projects) {
+            const project = this.rushConfiguration.getProjectByName(projectName);
+
+            shelljs.cd(project.projectFolder);
+            this.consoleBox(`Package name: ${projectName}`);
+
+            const result = shelljs.exec(command);
+
+            if (result.code === 0) {
+                succeededProjects.push(projectName);
+            } else {
+                failedProjects.push(projectName);
+            }
+        }
+
+        if (succeededProjects.length > 0) {
+            this.consoleBox(`Succeeded projects ${succeededProjects.length}`);
+            for (const project of succeededProjects) {
+                this.consoleWrite(project);
+            }
+        }
+
+        if (failedProjects.length > 0) {
+            this.consoleBox(`Failed projects ${failedProjects.length}`);
+            for (const project of failedProjects) {
+                this.consoleWrite(project);
+            }
+            process.exit(1);
+        }
+    }
+
+    private consoleWrite(...text: string[]): void {
```

Comment:
At first it was writing to console with a prefix, later on I removed it.
Fixed: https://github.com/SimplrJS/react-forms/commit/479b2962a32ff9177c364dac425ed8a21ed4a6ee